.env.example

```diff
@@ -0,0 +1,7 @@
+REACT_APP_ELASTICSEARCH_HOST=http://localhost:9200
```

Comment:
Is there a better way to provide these variables than to store them in a local file?

Reply:
@bmukheja This is the standard we normally follow.
But we can provide these values through various options like 
1) URL params
2) Through API returning all the settings.
3) Via settings table in DB but currently, we don't have any database.

Reply:
I think URL params should be OK for all, as we are already following that. 
@ronakmshah - But for Google api key, we need to make sure that the key is coming from the customer in production and not the same key as dev environment. DO you think VSD guys can help us in that?